docs/FeatureManagement/Samples/TargetingFilter.modified.sample.json

```diff
@@ -0,0 +1,60 @@
+{
+    "feature_management": {
+        "feature_flags": [
+            {
+                "id": "ComplexTargeting",
+                "description": "A feature flag using a targeting filter, that will return true for Alice, Stage1, and 50% of Stage2, and false for Dave and Stage3. The default rollout percentage is 25%.",
+                "enabled": true,
+                "conditions": {
+                    "client_filters": [
+                        {
+                            "name": "Microsoft.Targeting",
+                            "parameters": {
+                                "Audience": {
+                                    "Users": [
+                                        "Alice"
+                                    ],
+                                    "Groups": [
+                                        {
+                                            "Name": "Stage1",
+                                            "RolloutPercentage": 100
+                                        },
+                                        {
+                                            "Name": "Stage2",
+                                            "RolloutPercentage": 50
+                                        }
+                                    ],
+                                    "DefaultRolloutPercentage": 25,
+                                    "Exclusion": {
+                                        "Users": ["Dave"],
+                                        "Groups": ["Stage3"]
+                                    }
+                                }
+                            }
+                        }
+                    ]
+                }
+            },
+            {
+                "id": "RolloutPercentageUpdate",
```

Comment:
I am thinking this feature flag is not very easy to follow for those people who don't have the context. 
Currently, the ComplexTargeting flag in TargetingFilter.modified is not used. And for people have no context, they may feel strange why we have TargetingFilter and TargetingFilter.modified. And the testcases for Brittney locate in two places.

I still think, instead of having two feature flags both named "RolloutPercentageUpdate" in two files, it would be better to have two feature flags: "RolloutPercentage61" and "RolloutPercentage62" in one file. And we should put all testcases around them in one file.

Reply:
The idea is you don't need to understand these tests in order to validate the feature management libraries. And this is a really important check. It's validating that the algorithm we are using matches across all of the libraries.

We **CAN'T** change the name as that would change the targeting of the rollout and ruin the whole point of the test. The idea is that as 61% we have a user that isn't part of the rollout but at 62% they are. That will not be the case if the name of the feature flag changes.